client/components/Landing.tsx

```diff
@@ -1,12 +1,13 @@
-import { Image, Heading, Container, Text, Link } from "@chakra-ui/react";
+import { Image, Heading, Box, Text, Link } from "@chakra-ui/react";
 
+import NextLink from 'next/link'
 
-import { ButtonRed, ButtonGreen, ButtonBlue } from "../styles/ui-components/Chakra-Button"
+import { ButtonRed, ButtonBlack, ButtonGreen, ButtonBlue } from "../styles/ui-components/Chakra-Button"
 
 export default function Landing() {
   return (
-    <Container>
-      <Container
+    <Box>
```

Comment:
Curious why the switch from `Container` to `Box` ?

Reply:
Container is mostly for content with text, as it centers paragraphs and limits to 60ch. 

Box is a general `div` pretty much anything (no pre-set values as far as I know)

[Container](https://chakra-ui.com/docs/layout/container)

[Box](https://chakra-ui.com/docs/layout/box)